### PR TITLE
Update email-stuck-queues.md

### PR DESCRIPTION
**Pull request by Kovai-MSFT**
Updated the article with aspects of Hot Recipients and SRP limit changes which can potentially lead to queue build up at On-premises end.

### CODE REVIEW

File: Exchange/ExchangeServer/mailflow/email-stuck-queues.md [hunk 1]
```diff
@@ -24,11 +24,11 @@ appliesto:
...
+In Microsoft Exchange Server 2019, 2016, or 2013, email messages may be stuck in on-premises message queues for several minutes if the server is configured to send to a single destination, such as Exchange Online. There are few or no deferrals (400-series SMTP response codes) from Exchange Online to account for the number of messages in the queue. This is inclusive of scenarios wherein mailflow from on-premises exceed Exchange Online limits such as [Receiving Limits,Sender-Recipient pair limits](https://docs.microsoft.com/en-us/office365/servicedescriptions/exchange-online-service-description/exchange-online-limits#receiving-limits) and Exchange Online returns NDR status codes 5.2.121, 5.2.122 back to on-premises, followed by 400-series SMTP response codes.
```
Comment (stanalek) [inline Exchange/ExchangeServer/mailflow/email-stuck-queues.md:27]: 1. I would keep English simple for sake of making it easy for non-native speakers and auto-translation tools.
2. SRP should still be defined here as it is used as "SRP" later on.

**Review by stanalek — APPROVED** [review]

File: Exchange/ExchangeServer/mailflow/email-stuck-queues.md (hunk 1 above)
Comment (Kovai-MSFT) [reply Exchange/ExchangeServer/mailflow/email-stuck-queues.md:27]: Let's hold off on this for now based on Michelle's comments

Comment (stanalek) [reply Exchange/ExchangeServer/mailflow/email-stuck-queues.md:27]: Ok, not sure what I need to do on this GitHub review

**Review by stanalek — CHANGES_REQUESTED** [review]
Putting this one on hold for now.

**Pull request closed without merging**